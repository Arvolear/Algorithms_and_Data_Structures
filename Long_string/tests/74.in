7672161912449723827212490261573
515898700271195932154766928513966