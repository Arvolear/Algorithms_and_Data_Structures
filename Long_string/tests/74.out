523570862183645655981979418775539
508226538358746208327554438252393
3958058358902985439307050540423347994308428198626664427523628518
67